<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scTouv</class>
 <widget class="QWidget" name="scTouv">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Houdini</string>
  </property>
  <property name="styleSheet">
   <string notr="true">);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SizeConstraint::SetFixedSize</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../scTouc.qrc">:/scTouv/houdini_tool/img/SCtoUV.png</pixmap>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="leftMargin">
      <number>6</number>
     </property>
     <property name="topMargin">
      <number>6</number>
     </property>
     <property name="rightMargin">
      <number>6</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>ASSET</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtPath">
         <property name="placeholderText">
          <string>geo path</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnImport">
         <property name="toolTip">
          <string>import geo to reto</string>
         </property>
         <property name="text">
          <string>Import geo</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="verticalWidget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="lblQuad">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>img/quad_remesh.jpg</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Quad Total Count</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignRight">
         <widget class="QLineEdit" name="editQuad"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QLabel" name="lblExport">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>10</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>EXPORT</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnUsd">
         <property name="text">
          <string>USD</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAbc">
         <property name="text">
          <string>ABC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnFbx">
         <property name="text">
          <string>FBX</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../scTouc.qrc"/>
 </resources>
 <connections/>
</ui>
